src/server/js/resources/rooms/model.js

```diff
@@ -18,6 +18,24 @@ const endpoints = {
     getAllRoomsByHouse: houseId => {
         return RoomModel.find({ house: houseId }).exec();
     },
+    createRoom: (root, args) => {
```

Comment:
you aren't using 'root' in any of these functions (i don't really even know what 'root' is) - you could remove it from the arguments!